composer.json

```diff
@@ -5,20 +5,13 @@
   "prefer-stable": true,
   "minimum-stability": "dev",
   "require": {
-    "pimcore/pimcore": "^10.6 || ^11.0",
-    "symfony/config": "^5.3 || ^6.1",
```

Comment:
I think you shouldn't remove directly used symfony components

Reply:
I removed them because they are required by pimcore anyways ...

Reply:
I will bump the components to 6.2 (same version as pimcore/pimcore)

Reply:
done with https://github.com/pimcore/output-data-config-toolkit/pull/83/commits/035fbaf56dfcf9ff9beb1764983944a0d977d62d